src/autopas/containers/linkedCells/traversals/SlicedBlkTraversal.h

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file SlicedBlkTraversal.h
+ *
+ * @date 08 Apr 2020
+ * @author gratl
+ *
+ * adapted by henkel to use the sli_blk traversal
+ */
+
+#pragma once
+
+#include <algorithm>
+
+#include "LinkedCellTraversalInterface.h"
+#include "autopas/containers/cellPairTraversals/SlicedBlkBasedTraversal.h"
+#include "autopas/containers/linkedCells/traversals/C08CellHandler.h"
+#include "autopas/containers/verletListsCellBased/verletListsCells/traversals/VerletListsCellsTraversal.h"
+#include "autopas/utils/ThreeDimensionalMapping.h"
+#include "autopas/utils/WrapOpenMP.h"
+
+namespace autopas {
+
+/**
+ * This class provides the multiple dimension sliced traversal.
+ *
+ * @tparam ParticleCell
+ * @tparam PairwiseFunctor
+ * @tparam dataLayout
+ * @tparam useNewton3
+ */
+template<class ParticleCell, class PairwiseFunctor, DataLayoutOption::Value dataLayout, bool useNewton3>
+class SlicedBlkTraversal : public SlicedBlkBasedTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewton3>,
```

Comment:
What's the reason for splitting this traversal in a SlicedBlkBases and the real one? Do you plan to implement further versions that work on the same base? If no merge the two files.